src/config/absinthe.ts

```diff
@@ -49,8 +49,11 @@ socketConnection.onError(async (error: any) => {
   if (connectionFailureCounter >= CONNECTION_RECONNECT_ATTEMPTS) {
     socketConnection.disconnect();
 
-    // let's trigger the reconnect function after 5sec
-    setTimeout(() => resetWSConnection(socketConnection), 5000);
+    // lets not indefinitely try to reconnect
+    if (connectionFailureCounter < WEBSOCKET_RECONNECTION_ATTEMPTS) {
```

Comment:
@mdshamoon 

We only attempt to connect  `CONNECTION_RECONNECT_ATTEMPTS` times, ie. 5 times. So not sure why we need this.

If this is not linked to floweditor changes maybe move these changes to separate PR.

Reply:
The condition we put for the CONNECTION_RECONNECT_ATTEMPTS does not stop reconnecting the WebSocket connection, it goes on indefinitely. 

I will create a separate issue and PR for this since these changes are not related to flow editor 
I added in this PR because we were getting logged out in the preview due to this. We can discuss this further in the call.